static/scripts/teamMembers.js

```diff
@@ -190,14 +190,14 @@ $(document).ready(() => {
 			type: 'GET',
 			url: `${window.location.origin}/schools`,
 			data: {
-				$limit: false,
 				federalState,
-				hideOwnSchool: true,
 			},
 		}).done((schools) => {
 			const schoolSelect = $('#school');
 			schoolSelect.find('option').remove();
 			schools.forEach((school) => {
+				// TODO: Do we really want to only filter out expert schools here?
+				// What about schools with purpose "test" or "tombstone" or else?
```

Comment:
I am unsure. If we filter test school, that we break the functinallity for testing it by us. But tombstone should be filtered too.
But it sound like a logic that should be handled by the backend, or?

Reply:
Yep, I also think this should be in the backend. I'll apply the filter there.